src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntegerTests.cs

```diff
@@ -14687,6 +14738,92 @@ enum E { }
             }
         }
 
+        [Fact, WorkItem(60714, "https://github.com/dotnet/roslyn/issues/60714")]
+        public void ConversionFromNuintToDouble()
+        {
+            var source = """
+if (!System.Environment.Is64BitProcess)
```

Comment:
Is there an equivalent test that fails on 32-bit platforms? 

For instance, consider changing the code below to:
```csharp
nuint x = System.IntPtr.Size == 4 ?
    uint.MaxValue :
    unchecked((nuint)ulong.MaxValue);
ulong y = System.IntPtr.Size == 4 ?
    uint.MaxValue :
    ulong.MaxValue;
```

Reply:
Updated and verified those two tests fails both on 32-bit and 64-bit platforms prior to fix.